README.md

```diff
@@ -976,18 +945,17 @@ documentation](https://docs.rubocop.org/rubocop/configuration.html#inheriting-co
   puts "$global = #{$global}"
   ~~~
 
-* Don't use `Object#to_s` on interpolated objects. It's invoked on them
-  automatically.
+* Avoid `Object#to_s` on interpolated objects.
```

Comment:
Imo that's another `don't` case because it's directly what it does, I can't imagine needing this in any scenario, but maybe it's my imagination